src/router/routes.js

```diff
@@ -101,9 +101,9 @@ export default [
 	{
 		path: '/lp/:dynamicRoute',
 		component: () => import('@/pages/ContentfulPage'),
-		meta: route => ({
```

Comment:
I guess you can't do this. 

This was casually working because if meta is undefined,  `contentfulPage` will actually make the pageQuery with an empty key, which will return all routes from contentful sorted by date added, and because this was the latest page on contentful it was being  rendered. But when a new route was added this morning (by mobile), it completely broke the landing page. 

Not sure if we should validate the meta param in contentfulPage to handle this...

Reply:
I think we should either provide a default OR probably just redirect to home if the key is empty.